### DISCUSSION TITLE
Repository topics are missing some major topics that are found on other graph vis libs

### DISCUSSION THREAD

**Discussion by TannerW** [reaviz/reagraph#106] [Ideas]
Hi, 
Love what is going on here and I'd like to help its visibility. I wanted to recommend adding the following github topics to the repo's About section:

- [#graph-drawing](https://github.com/topics/graph-drawing)
- [#network-visualization](https://github.com/topics/network-visualization)
- [#graph-visualization](https://github.com/topics/graph-visualization)
- [#knowledge-graph](https://github.com/topics/knowledge-graph)

**Comment by amcdnl** [comment]
Thanks for the recommendation - added :D
